Backend/CarRentalApp/CarRentalApp/Program.cs

```diff
@@ -1,23 +1,65 @@
-var builder = WebApplication.CreateBuilder(args);
+using CarRentalApp.Configuration.JWT.Access;
+using Microsoft.AspNetCore.Authentication.JwtBearer;
+using Microsoft.IdentityModel.Tokens;
+using System.Text;
+using CarRentalApp.Services.Identity;
+using CarRentalApp.Services.Token;
+using CarRentalApp.Services.Authentication;
+using CarRentalApp.Services.Data;
+using CarRentalApp.Models.Contexts;
+using Microsoft.EntityFrameworkCore;
 
-// Add services to the container.
+var builder = WebApplication.CreateBuilder(args);
 
 builder.Services.AddControllers();
 
-builder.Services.AddEndpointsApiExplorer();
+var configurationString = builder.Configuration.GetConnectionString("sqlserver");
+builder.Services.AddDbContext<AuthenticationDbContext>(options => options.UseSqlServer(configurationString));
 
-var app = builder.Build();
+builder.Services.AddScoped<UserRepository>();
+builder.Services.AddScoped<UserService>();
+
+builder.Services.AddScoped<PasswordService, ShaPasswordService>();
+
+builder.Services.AddScoped<TokenService>();
+
+builder.Services.AddScoped<AuthenticationService>();
+
+builder.Services.Configure<AccessJwtConfig>(
+    builder.Configuration.GetSection(AccessJwtConfig.Section)
+);
 
-// Configure the HTTP request pipeline.
-if (app.Environment.IsDevelopment())
+var accessJwtConfig = new AccessJwtConfig();
+builder.Configuration.Bind(AccessJwtConfig.Section, accessJwtConfig);
+
+var accessJwtValidationParams = accessJwtConfig.ValidationParameters;
+accessJwtValidationParams.IssuerSigningKey = new SymmetricSecurityKey(
+    Encoding.UTF8.GetBytes(accessJwtConfig.GenerationParameters.Secret)
+);
+
+builder.Services.AddAuthentication(JwtBearerDefaults.AuthenticationScheme).AddJwtBearer(options =>
+{
+    options.TokenValidationParameters = accessJwtValidationParams;
+});
+
+builder.Services.AddCors(options =>
 {
-    
-}
+    options.AddDefaultPolicy(
+        builder =>
+        {
+            builder.AllowAnyOrigin()
+                .AllowAnyMethod()
+                .AllowAnyHeader();
+        });
```

Comment:
```suggestion
    options.AddDefaultPolicy(
        builder =>
        {
            builder
                .AllowAnyOrigin()
                .AllowAnyMethod()
                .AllowAnyHeader();
        }
    );
```

Reply:
Suggested code style:
- place ALL chained method calls on the same level;
- place closing bracket on the same level with the statement that contains opening bracket;